domain/cloud/state/state.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/juju/utils/v3"
 
 	"github.com/juju/juju/cloud"
+	coreDB "github.com/juju/juju/core/database"
```

Comment:
nit: Don't abbreviate when it's not required. Also, I'd rather keep all imports lowercase.

```suggestion
	coredatabase "github.com/juju/juju/core/database"
```

Reply:
done